File: CodeForces/Python/CF_136A_Presents.py
# Problem: 136A. Presents
# Link: https://codeforces.com/problemset/problem/136/A

people = int(input())

arr = [0] * people

places = list(map(int, input().split()))

for i in range(1, people + 1):
  place = places[i - 1]
  arr[place - 1] = i

for val in arr:
  print(val, end=" ")
print()

File: CodeForces/Java/CF_136A_Presents.java
//  Problem: 136A. Presents
//  Link: https://codeforces.com/problemset/problem/136/A

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    int people = scan.nextInt();
    int[] arr = new int[people];
    
    for (int i =1; i<=people;i++){
      int place = scan.nextInt();
      arr[place-1] = i;
    }
    scan.close();
    
    for (int i =0; i<arr.length;i++){
      System.out.print(arr[i] + " ");
    }
  }
}
